client-templates/bitbucket-server/accept.json.sample

```diff
@@ -526,6 +524,17 @@
         "password": "${BITBUCKET_PASSWORD}"
       }
     },
+    {
+      "//": "get list of API capabilities",
+      "method": "GET",
+      "path": "/rest/capabilities",
+      "origin": "https://${BITBUCKET}",
+      "auth": {
+        "scheme": "basic",
+        "username": "${BITBUCKET_USERNAME}",
+        "password": "${BITBUCKET_PASSWORD}"
+      }
+    },
     {
```

Comment:
what about the one at: https://github.com/snyk/bitbucket-server-agent/blob/develop/lib/bitbucket-server/codeinsights.js#L109? `${this.url}/rest/insights/latest/capabilities`

Reply:
[Line 673](https://github.com/snyk/broker/pull/194/files#diff-e013066c9a35159b140d2c307505b7b1R673)